intelmq/bots/parsers/bambenek/parser_dgafeed.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+See docs/Feeds.md
+"""
+
+import sys
+
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.message import Event
+
+
+class BambenekDGAfeedParserBot(ParserBot):
+
+    def parse_line(self, line, report):
+        if line.startswith("#"):
+            self.tempdata.append(line)
+        else:
+            lvalue = line.split(',')
+            event = Event(report)
+
+            event.add('source.fqdn', lvalue[0])
+            event.add('event_description.text', lvalue[1])
+            event.add('time.source', lvalue[2] + " 00:00 UTC")
+            event.add('event_description.url', lvalue[3])
+            event.add('classification.type', 'ransomware')
```

Comment:
Should I change the classification.type to "dga domain" and also make the needed changes to harmonization.py to allow dga domain and also map it to Malicious Code in the Taxonomy?